R/DBI-pool.R

```diff
@@ -47,8 +48,39 @@ setMethod("onDestroy", "DBIConnection", function(object) {
 #' @export
 #' @rdname object
 setMethod("onValidate", "DBIConnection", function(object) {
-  check <- dbGetQuery(object, "SELECT 1")
-  df <- data.frame(1)
-  names(df) <- "1"
-  (check == df)[1,]
+  pool <- attr(object, "..metadata", exact = TRUE)$..pool
```

Comment:
No, the purpose of a `pool_` prefix would not be as an adjective, but as a namespace (to prevent collision with other packages' interpretation of attributes). So `pool` in this case is the package name.